src/components/balances/AssetsTable/index.tsx

```diff
@@ -13,38 +13,46 @@ import useIsSafeOwner from '@/hooks/useIsSafeOwner'
 import Track from '@/components/common/Track'
 import { ASSETS_EVENTS } from '@/services/analytics/events/assets'
 import InfoIcon from '@/public/images/notifications/info.svg'
+import useIsMobile from '@/hooks/useIsMobile'
 
 interface AssetsTableProps {
   items?: SafeBalanceResponse['items']
 }
 
-const headCells = [
-  {
-    id: 'asset',
-    label: 'Asset',
-    width: '60%',
-  },
-  {
-    id: 'balance',
-    label: 'Balance',
-    width: '20%',
-  },
-  {
-    id: 'value',
-    label: 'Value',
-    width: '20%',
-  },
-  {
-    id: 'actions',
-    label: '',
-    width: '20%',
-    sticky: true,
-  },
-]
-
 const AssetsTable = ({ items }: AssetsTableProps): ReactElement => {
   const [selectedAsset, setSelectedAsset] = useState<string | undefined>()
   const isSafeOwner = useIsSafeOwner()
+  const isMobile = useIsMobile()
```

Comment:
Can't you just hide the last column via CSS?

Reply:
There are [two dependent flags for hiding the actions](https://github.com/safe-global/web-core/blob/improve-enhanced-tables/src/components/balances/AssetsTable/index.tsx#L24-L27) so I thought keeping them together made the most sense. The `EnhancedTable` then hides it via CSS [here](https://github.com/safe-global/web-core/blob/improve-enhanced-tables/src/components/common/EnhancedTable/index.tsx#L77) and [here](https://github.com/safe-global/web-core/blob/improve-enhanced-tables/src/components/common/EnhancedTable/index.tsx#L144).

Reply:
They are not dependent. A user can be a non-owner on both desktop and mobile.